HopsanGUI/Widgets/ModelWidget.cpp

```diff
@@ -945,6 +945,7 @@ void ModelWidget::openCurrentContainerInNewTab()
             pContainer->setModelFile("");
             lockModelEditingFull(true);
             mpExternalSystemWarningWidget->setVisible(false);
+            pContainer->setSubTypeName("");
```

Comment:
I have not looked deep into this problem. But it feels like it would be better to do this in the code that loads and embeds the subsystem. Not here in the code that seems to open it in a new tab. But maybe I need to look deeper.

Reply:
That is actually this code. The name of the function is a bit weird though. As far as I can see it does not open any new tab. It just enables editing, sets the model file to an empty string and hides the external system widget.

Did we at some point change the behavior from opening external models in a new tab to embedding them in the model?